build/gulp/tasks/screener.ts

```diff
@@ -10,6 +11,9 @@ const { paths } = config
 // ----------------------------------------
 
 task('screener:runner', cb => {
+  // screener-runner doesn't allow to pass custom options
+  if (argv.filter) process.env.SCREENER_FILTER = argv.filter
```

Comment:
just to be sure - isn't it possible to pass argument here and capture it in `screener.config` via `argv`?

Reply:
Exactly 😭 

```
yarn run v1.13.0
$ C:\Users\olfedias\WebstormProjects\react\node_modules\.bin\screener-runner --filter

  error: unknown option `--filter'

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```